### DISCUSSION TITLE
Query Achievements

### DISCUSSION THREAD

**Discussion by CoDaJac** [legendary-gl/legendary#161] [Q&A]
Hi,
thanks for all the work you put into the project. 
The documentation of epic is unfortunately extremely bad (for web-requests there is actually none), but with your project accessing their API without the Epic SDK is a lot easier.

To my question:
Is it currently not possible to query Achievements at all or have you not implemented this feature yet?

Thank you.

**Comment by derrod** [comment]
The Epic Games Launcher also doesn't have a way to see Achievements, so for now that's not really something I have looked into or considered implementing. Maybe in the future when legendary actually has a GUI that could display them.

**Comment by derrod** [comment]
Now having briefly looked into it; achievements aren't available via any of the Epic Games Launcher APIs by the looks of it, and can currently only be retrieved by the game itself.
